management_api.go

```diff
@@ -256,6 +254,6 @@ func (e *Enforcer) RemoveFilteredNamedGroupingPolicy(ptype string, fieldIndex in
 }
 
 // AddFunction adds a customized function.
-func (e *Enforcer) AddFunction(name string, function govaluate.ExpressionFunction) {
+func (e *Enforcer) AddFunction(name string, function func(args ...interface{}) (interface{}, error)) {
```

Comment:
Why do this?

Reply:
It seems to be for mock.

Reply:
The interface cannot automatically convert equal types.  The two function types are  same.Or we can also change the type in IEnforce

Reply:
I prefer `govaluate.ExpressionFunction` because it's easy to know where it comes from, even if it needs to import a lib.

Reply:
Ok i'll modify it right away

Reply:
Sorry. Does this change meaning that any user that wants to write a custom function has to import govaluate? If yes, please revert back.

Reply:
I don't want to cause any trouble for our users.

Reply:
Of course not, the compiler will only have problems with matching function type. It will be automatically corrected in parameter passing. Users only need to do the same as before.